src/Exporter.php

```diff
@@ -533,6 +561,31 @@ public function getStoreConfigValue($storeId, $configKey)
         return null;
     }
 
+    /**
+     *
+     *
+     * @return $this
+     */
+    protected function initLanguageUrlPrefix()
+    {
+        /** @var \Findologic\Plentymarkets\Parser\Stores $stores */
+        $stores = $this->getRegistry()->get('Stores');
+
+        $configurationLanguage = strtolower($this->getConfig()->getLanguage());
+        $urlPrefix = '';
+
+        if (
+            $configurationLanguage != $stores->getStoreDefaultLanguage($this->getConfig()->getMultishopId()) &&
```

Comment:
This is actually the wrong ID. Use `\Findologic\Plentymarkets\Exporter::getStorePlentyId` instead of `::getMultishopId`